assignments/callbacks.js

```diff
@@ -20,33 +18,70 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
   });
 
 */
+const items = [1, 2, 3, 4, 'hello'] //with 5 elements where count starts from 0 || 
+        function firstItem(arr, cb) {
+          return cb(arr[4]);  // consoles hello
+        }
+        firstItem(items, function(first) {
+          console.log(first)
+        });
 
-
+const numero = [1, 2, 3, 4, 'hello'] //with 5 elements where count starts from 0 || 
 function getLength(arr, cb) {
+  return cb(arr.length)
   // getLength passes the length of the array into the callback.
 }
+console.log ();
 
 function last(arr, cb) {
+     return cb(arr[arr.length -1])
   // last passes the last item of the array into the callback.
 }
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+//   let a = 3
+// function addTwo(x) {
+//   let ret = x + 2
+//   return ret
+//  }
+//  let b = addTwo(a)
+//  console.log(b);
+
+  return cb(arr[x + y]);
```

Comment:
You want to return a callback that will add two numbers here, what You are doing is getting index of `arr` which is not even passed as an argument or defined inside the function.